content/docs/1.5.0/references/settings.md

```diff
@@ -7,6 +7,7 @@ weight: 1
 - [General](#general)
   - [Node Drain Policy](#node-drain-policy)
   - [Automatically Cleanup System Generated Snapshot](#automatically-cleanup-system-generated-snapshot)
+  - [Automatically Cleanup Recurring Job Backup Snapshot](#automatically-cleanup-recurring-job-backup-snapshot)
```

Comment:
Suggested wording: Automatically Clean Up Outdated Snapshots of Recurring Backup Jobs

Reply:
Moved to 1.6.0 and Modified. Thanks for suggestions.